src/compute/grid/track_sizing.rs

```diff
@@ -566,6 +600,12 @@ fn distribute_item_space_to_base_size(
             return;
         }
 
+        // 0. Prep limit function
```

Comment:
This could use a bit more description; the nature and value of this step is fairly unclear when reading the code.

Reply:
Description added!